### PR TITLE
main.go: fix help text for --unsafe-passthrough-paths

### PR DESCRIPTION
**Pull request by simonpasquier**
Signed-off-by: Simon Pasquier <spasquie@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,8 +103,7 @@ This is enforced for any case, whether a label matcher is specified in the origi
 
 Similar to query endpoint, for metadata endpoints `/api/v1/series`, `/api/v1/labels`, `/api/v1/label/<name>/values` the proxy injects the specified label all the provided `match[]` selectors.
 
-NOTE: At the moment of creation `/api/v1/labels`, `/api/v1/label/<name>/values` does not support `match[]` so they are disabled by default. Use `-enable-label-apis` flag to enable
-those (see https://github.com/prometheus/prometheus/issues/6178 for tracking development).
+NOTE: When the `/api/v1/labels` and `/api/v1/label/<name>/values` endpoints were added to `prom-label-proxy`, the Prometheus and Thanos endpoints didn't support the `match[]` parameter hence the `prom-label-proxy` labels endpoints are disabled by default. Use the `-enable-label-apis` flag to enable after you've ensured that the upstream endpoints support label selectors (see https://github.com/prometheus/prometheus/issues/6178 and https://github.com/thanos-io/thanos/issues/3351).
```
Comment (yeya24) [inline README.md:106]: Thanos supports label matchers already in https://github.com/thanos-io/thanos/pull/3566, just implementing it in a not so efficient way (via Series).
Now efficient labels matchers pushdown is supported too in the latest Thanos release (v0.23.0)

Comment (simonpasquier) [reply README.md:106]: Thanks! Do you recommend that we mention Thanos >= v0.23.0 explicitly here?

**Review by squat — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (squat) [reply README.md:106]: ping @yeya24 :))

Comment (jan--f) [reply README.md:106]: Seems like the the feature was added in [Thanos v0.18.0](https://github.com/thanos-io/thanos/releases/tag/v0.18.0)

Comment (simonpasquier) [reply README.md]: I've clarified the text, mentioning version numbers instead of pull requests IDs.

**Comment by jan--f** [comment]
Great, thank you! @squat feel free to merge this, I'll adjust #88 accordingly.

**Review by squat — APPROVED** [review]

**Pull request merged**